digitalocean/droplet/resource_droplet.go

```diff
@@ -580,6 +652,27 @@ func resourceDigitalOceanDropletUpdate(ctx context.Context, d *schema.ResourceDa
 		}
 	}
 
+	if d.HasChange("backup_policy") {
+		var newPolicy godo.DropletBackupPolicyRequest
+		bp, ok := d.GetOk("backup_policy")
+		if ok && bp != nil { // backup_policy must be set.
+			_, new := d.GetChange("backup_policy")
+			if newList, ok := new.([]interface{}); ok && len(newList) != 0 {
+				newPolicy = new.([]interface{})[0].(godo.DropletBackupPolicyRequest)
+			}
+
+			action, _, err := client.DropletActions.ChangeBackupPolicy(context.Background(), id, &newPolicy)
```

Comment:
This will fail if backups have not already been enabled. We should think through the interaction between the new `backup_policy` attribute and the existing `backups` one.   For example, what do we do if `backups = false` but a `backup_policy` is specified?

Reply:
updated the logic here: [7b40f82](https://github.com/digitalocean/terraform-provider-digitalocean/pull/1261/commits/7b40f82463891fded1da513640835b50136f6146)